MekHQ/src/mekhq/campaign/mission/enums/AtBContractType.java

```diff
@@ -351,97 +351,97 @@ public int generateEventType(Campaign campaign) {
      *
      * @return an integer representing the event type.
      */
-    public int generateStratConEvent() {
+    public AtBEventType generateStratConEvent() {
         final int roll = Compute.randomInt(20) + 1;
```

Comment:
this really should be a static map or something like it.

Reply:
Thanks, that probably does make sense. I'm guessing we'd need to simplify the rolls to avoid a bunch of duplicate values (e.g., for keys 10-13 and 14-15), right?

By the way, with the current direction I'm heading with the Contract classes, this